join-welcome/handler.py

```diff
@@ -6,71 +6,81 @@
 import hmac
 from time import perf_counter
 
-import requests
-
-def handle(req):
-    """handle a request to the function
-    Args:
-        req (str): request body
-    """
-
-    log_event(req)
+def handle(event, context):
+    SLACK_SIG_HEADER = "X-Slack-Signature"
+    SLACK_TIMESTAMP_HEADER = "X-Slack-Request-Timestamp"
+    
+    log_event(event.body)
```

Comment:
Is event.body JSON or the raw value? It will affect the hmac results

Reply:
Nope its a single bytestring.
http://flask.pocoo.org/docs/1.0/api/#flask.Request.get_data